pkg/cmd/root/sysinfo_freebsd.go

```diff
@@ -3,13 +3,15 @@
 package root
 
 import (
+	"fmt"
+
```

Comment:
And here?

Reply:
Go linter automatically puts the new line